resources/deletedb.yaml

```diff
@@ -1,3 +1,45 @@
+---
```

Comment:
these serviceaccounts are almost identical: deletedb and hatest for example, for simplicity sake can we shrink most of them to one for stolon?